cross/epic5/patches/config.cache.patch

```diff
@@ -0,0 +1,4 @@
+--- /dev/null     2011-08-22 20:38:12.000000000 +0200
++++ config.cache  2011-12-16 23:15:45.000000000 +0100
+@@ -0,0 +1 @@
++ac_cv_func_getpgrp_void=yes
```

Comment:
Any idea how? I couldn't figure out a way, but it does work as an environment variable, see the most recent push.